app/models/monthly_report.rb

```diff
@@ -99,6 +99,20 @@ def browseable?(other_user)
     shipped? || user == other_user
   end
 
+  def self.target_month_select_options
+    return [] if MonthlyReport.released.blank?
+    first_month = MonthlyReport.released.minimum(:target_month)
+    last_month = MonthlyReport.released.maximum(:target_month)
+    all_months(first_month, last_month).map{ |month| [month.strftime('%Y年%m月'), month] }
```

Comment:
__[rubocop]__
* Space missing to the left of {.

<!-- Comment by [SideCI](https://sideci.com) -->